app/css/vendors/ip-camera.css

```diff
@@ -0,0 +1,77 @@
+.camera-view {
+  display: flex;
+  flex-direction: column;
+}
+
+.camera-view__body {
```

Comment:
I  don't usually have opinions on CSS. If you think that's the best way to do, implement it, update the existing styles and document the best practices in `app/css/README.md`.